ngraph/core/include/ngraph/pass/low_latency.hpp

```diff
@@ -13,43 +13,47 @@ namespace ngraph
 {
     namespace pass
     {
-        class NGRAPH_API LowLatency;
+        /**
+         * @brief The transformation finds all TensorIterator layers in the network, processes all
+         * back
```

Comment:
Can you correct the line breaks in this documentation block? The one in the old diff was more consistent and after the change we have some very short lines with just one or only a couple of words.

Reply:
Fixed